controllers/buildpipeline/buildpipeline_controller.go

```diff
@@ -104,7 +106,23 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 	if err != nil {
 		logger.Error(err, "Failed to get Application from Component",
 			"Component.Name ", component.Name, "Component.Namespace ", component.Namespace)
-		return ctrl.Result{}, err
+		msg := fmt.Sprint("Failed to get Application from Component",
```

Comment:
I don't understand this Sprint, looks like log.

Reply:
Updated.